pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "elotl"
+version = "0.0.17"
+description = "Python package for Natural Language Processing (NLP), focused on low-resource languages spoken in Mexico."
+authors = [
+	"Paul Aguilar <paul.aguilar.enriquez@hotmail.com>",
+	"Robert Pugh <robertpugh408@gmail.com>",
+	"Diego A. Barriga Martínez <dbarriga@ciencias.unam.mx>"
+]
+maintainers = ["Robert Pugh <robertpugh408@gmail.com>"]
+license = "MLP-2.0"
+readme = "README.md"
+repository = "https://github.com/umoqnier/elotl_pkg"
```

Comment:
Is this supposed to point to the fork?

Reply:
Oh no, thanks. I will update this. Sorry :p

Reply:
Done!